roles/kubernetes-apps/ingress_controller/ingress_nginx/defaults/main.yml

```diff
@@ -3,6 +3,7 @@ ingress_nginx_namespace: "ingress-nginx"
 ingress_nginx_host_network: false
 ingress_nginx_nodeselector:
   node-role.kubernetes.io/master: "true"
+ingress_nginx_tolerations: true
```

Comment:
Should not default to true. We don't want the ingress controller on masters

Reply:
Ok. then I though ingress controller should be deployed on master node by following default.yaml:

    ingress_nginx_nodeselector:
       node-role.kubernetes.io/master: "true"

So you don't want the ingress controller on masters, then do you prefer following configuration?

```
ingress_nginx_host_network: false
-ingress_nginx_nodeselector:
-  node-role.kubernetes.io/master: "true"
-ingress_nginx_tolerations: true
+ingress_nginx_nodeselector:
+ingress_nginx_tolerations: false
```